client/views/teams/contextualBar/info/TeamsInfo.js

```diff
@@ -144,6 +151,13 @@ const TeamsInfo = ({
 							</InfoPanel.Field>
 						)}
 
+						{tags && tags.length !== 0 && (
```

Comment:
This one is a tricky one let me explain, a Team has a main channel, that is public for all the team members, but not for the outside, so I think users might not find them in discovery. I don't think we should change anything right now, but let's keep that in mind.